doc/whats-new.md

```diff
@@ -293,27 +307,27 @@ the engine that generated it, and some relevant parameters, like the
 investigation time in the case of the hazard outputs. In the future
 we may add even more metadata and extend the approach to other outputs.
 
-Before the release 3.3 the engine had the ability to associate site
-model parameters to hazard sites on a grid. Such feature was sometimes
+Before release 3.3, the engine had the ability to associate site
+model parameters to hazard sites on a grid. This feature was sometimes
 buggy and removed, by recommending to the users the command `oq
-prepare_site_model` instead.  `oq prepare_site_model` is able to
+prepare_site_model` instead. `oq prepare_site_model` is able to
 produce a `site_model.csv` file with sites on the grid and it performs the
-associations only once and for all.
+associations explicitly, once and for all.
 
 In this release, we restored the ability to perform the association
-directly in the engine.  This is less efficient than using `oq
-prepare_site_model`, since the same associations will be recomputed at
+directly in the engine. This is less efficient than using `oq
+prepare_site_model`, since the same associations will be recomputed during
 each run. It is still useful for people wanting to experiment with the
 grid spacing: they can run several calculations and when they are
 happy with the grid spacing, run `oq prepare_site_model` and fix the
-site model once and for all with the good grid spacing.
```

Comment:
The use case that isn't yet covered by `oq prepare_site_model` is the one where, say, the user has a rupture model and would like to compute ground motion fields on a grid around the rupture. The key inputs in such cases would be:
1. A `region` polygon, defined by lon lat co-ordinates of its vertices
2. A `region_grid_spacing`
3. A `site_model_file`

Reply:
In that case the user should generate a file with the hazard sites (by gridding the polygon) and pass it to `oq prepare_site_model` together with vs30.csv file:

`$ oq prepate_site_model -s sites.csv vs30.csv`